From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@JulioGold__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Commit:
task 629: create new CDN caching topics

Commit:
task 629: update TOC with new topics

Commit:
task 629: edits

Commit:
task 629: edits

Commit:
task 629: rename delivery rules to caching rules, add screen shot, incorp review comments

Commit:
task 629: reorg caching in TOC, edits

Commit:
task 629: reorg TOC, edits

Commit:
task 629: edit to match controls in CDN caching page

Commit:
task 629: additional minor edits

Commit:
Update table for caching rules

Commit:
task 629: additional edits

Commit:
task 629: additional edits

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-cdn

Commit:
edits for SEO

zjalexander committed:
Remove preview

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-cdn

Commit:
task 634: Add caching rules info

ktoliver committed:
edit pass: two Machine Learning articles

ktoliver committed:
Merge branch 'master' into using-git-ml-project

ktoliver committed:
fix merge conflict

ktoliver committed:
Merge branch 'using-git-ml-project' of https://github.com/ktoliver/azure-docs-pr into using-git-ml-project

GitHubber17 committed:
edit pass: ad dev guide, appmodel overview, includes

v-viinde committed:
USER STORY 1135553 SaaS App Tutorial: Trello Update

v-viinde committed:
Merge branch 'master' into trelloupdate

v-viinde committed:
Updated

v-viinde committed:
Merge branch 'trelloupdate' of https://github.com/v-viinde/azure-docs-pr into trelloupdate

v-viinde committed:
Update

v-viinde committed:
Updated

v-viinde committed:
Updated

v-viinde committed:
USER STORY 1137599 SaaS App Tutorial: SAML SSO for Bamboo by resolution GmbH

v-viinde committed:
USER STORY 1137598 SaaS App Tutorial: SAML SSO for Bitbucket by resolution GmbH

spelluru committed:
New article on improving performance for Azure-SSIS IR

spelluru committed:
fixed TOC issue

spelluru committed:
updates

spelluru committed:
review and edit

spelluru committed:
minor fix

spelluru committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into adfssisperf1201

spelluru committed:
one more fix

spelluru committed:
fixed broken links

v-viinde committed:
updated

v-viinde committed:
updated

v-viinde committed:
updated

v-viinde committed:
Updated

spelluru committed:
incremental copy tutorials

RicksterCDN committed:
fixing cloudinit toc entries

RicksterCDN committed:
fixing up old links and refs

spelluru committed:
updated TOC

spelluru committed:
include file

spelluru committed:
few more issues

nberdy committed:
Update about-iot-dps.md

Updating for GA

johnkemnetz committed:
Update monitoring-diagnostic-logs-schema.md

spelluru committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into inctutorials1204

v-viinde committed:
USER STORY 1139545 SaaS App Tutorial: IMAGE WORKS Update

v-viinde committed:
Merge branch 'master' into imageworksupdate

spelluru committed:
commiting

spelluru committed:
reviewed text, updated images

spelluru committed:
tested multi table copy tutorial

RicksterCDN committed:
removed accidental file

Commit:
rajat01

nberdy committed:
Update about-iot-dps.md

MikeDodaro committed:
New file for Autosuggest endpoint summary. Added link to TOC.

rjagiewich committed:
Merge pull request #29207 from RicksterCDN/cloudinit-toc

Cloudinit toc

GitHubber17 committed:
Merge pull request #29228 from nberdy/patch-1

Update about-iot-dps.md

spelluru committed:
reviewed and updated

spelluru committed:
Updated

Commit:
freshness

MikeDodaro committed:
typo: Video search -> Autosuggest

jonbeck7 committed:
Update sizes.md

jonbeck7 committed:
Update sizes.md

neilpeterson committed:
updated metadata

PRMerger17 committed:
Merge pull request #29297 from Blackmist/freshness

freshness

jonbeck7 committed:
Update sizes.md

neilpeterson committed:
updated missing metadata

GitHubber17 committed:
Merge pull request #29066 from spelluru/adfssisperf1201

New article on improving performance for Azure-SSIS IR

sanjeev3 committed:
Update video-moderation-api.md

alkohli committed:
Remove the offline warning for the clone

sanjeev3 committed:
Update video-moderation-api.md

sanjeev3 committed:
Update video-moderation-api.md

carolinacmoravia committed:
M16776:  Text incorrectly locked as a tag: "data fac"

Hello, @spelluru,
Translator has reported source content issue that should be localized. We need to remove the ´´ and replace them by ** for bold type.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

PRMerger17 committed:
Merge pull request #29307 from alkohli/patch-159

Remove the offline warning for the clone

PRMerger15 committed:
Merge pull request #29308 from carolinacmoravia/patch-499

M16776:  Text incorrectly locked as a tag: "data fac"

ggailey777 committed:
Fixed link.

Commit:
Merge pull request #29301 from jonbeck7/patch-10

Update sizes.md

Commit:
Merge pull request #29303 from jonbeck7/patch-11

Update sizes.md

Commit:
Merge pull request #29300 from jonbeck7/patch-9

Update sizes.md

cherylmc committed:
classic

Commit:
Fix internal link

Commit:
Removing a note per engineering conversation

Commit:
Merge pull request #29287 from MarkusVi/rajat01

rajat01

ranvijaykumar committed:
Fixed typos

PRMerger14 committed:
Merge pull request #29314 from Blackmist/freshness

Removing a note per engineering conversation

PRMerger15 committed:
Merge pull request #29309 from sanjeev3/master

Minor fixes and updates.

Commit:
Merge pull request #29316 from ranvijaykumar/patch-4

Fixed typos

spelluru committed:
added steps for defining variables

ktoliver committed:
Merge branch 'master' into using-git-ml-project

Commit:
Merge pull request #29313 from cherylmc/c2

classic

cherylmc committed:
blade

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into c2

PRMerger15 committed:
Merge pull request #29311 from MikeWasson/patch-1

Fix internal link

MikeDodaro committed:
New file for Custom Search endpoint.  Added file to TOC.

Commit:
Merge branch 'master' of https://github.com/dksimpson/azure-docs-pr into dean-629

Commit:
Merge pull request #29325 from cherylmc/c2

C2

Commit:
Merge pull request #29319 from spelluru/adfspark1205

added steps for defining variables

spelluru committed:
unnecessary intro

MikeDodaro committed:
New file for Entity Search endpoint. Added to TOC.

PRMerger19 committed:
Merge pull request #27806 from zjalexander/zachal-sccm

Remove preview from SCCM content

CarlRabeler committed:
new graphic

MikeDodaro committed:
Typo Autosuggest -> Custom Search

MicrosoftGuyJFlo committed:
Add error message for globally banned passwords to docs

MikeDodaro committed:
Style fixes

Commit:
Fix merge issue in TOC - create Control caching behavior noce

GitHubber17 committed:
Merge pull request #29295 from MikeDodaro/autosuggest-endpoint

New file for Autosuggest endpoint summary. Added link to TOC.

PRMerger10 committed:
Merge pull request #29327 from spelluru/adfspark1205-2

unnecessary intro

MikeDodaro committed:
Spelling error

Ja-Dunn committed:
Merge pull request #29329 from CarlRabeler/danimir

new graphic

ShawnJackson committed:
Update using-git-ml-project.md

ShawnJackson committed:
Update roaming-and-collaboration.md

Ja-Dunn committed:
Merge pull request #26028 from dksimpson/dean-629

task 629: create new CDN caching topics

PRMerger10 committed:
Merge pull request #29251 from johnkemnetz/patch-1

Update monitoring-diagnostic-logs-schema.md

ShawnJackson committed:
Merge pull request #28589 from ktoliver/using-git-ml-project

edit pass: two Machine Learning articles

spelluru committed:
updated based feeedback

Ja-Dunn committed:
Merge pull request #29326 from MikeDodaro/endpoint-custom

New file for Custom Search endpoint.  Added file to TOC.

PRMerger20 committed:
Merge pull request #29310 from ggailey777/patch-3

Fixed link.

PRMerger14 committed:
Merge pull request #29330 from MicrosoftGuyJFlo/318273SSPR

Add error message for globally banned passwords to docs

ShawnJackson committed:
Update active-directory-appmodel-v2-overview.md

ShawnJackson committed:
Merge pull request #28626 from GitHubber17/edit-pass-tfs72155-dstrockis

edit pass: active directory topics

rmca14 committed:
Merge pull request #29302 from neilpeterson/acs-metadata-cleanup

updated metadata

Commit:
add caching rules info-fix merge issues

Lisaco88 committed:
Merge pull request #29331 from MikeDodaro/endpoint-entities

Endpoint entities

Commit:
updates to address content bug 1082921

PRMerger19 committed:
Merge pull request #29276 from v-viinde/imageworksupdate

USER STORY 1139545 SaaS App Tutorial: IMAGE WORKS Update

Ja-Dunn committed:
Merge pull request #29194 from spelluru/inctutorials1204

incremental copy tutorials

anilmur committed:
Added HEVC as an available output video codec

Ja-Dunn committed:
Merge pull request #29055 from v-viinde/bitbucket

USER STORY 1137598 SaaS App Tutorial: SAML SSO for Bitbucket by resol…

Ja-Dunn committed:
Merge pull request #29054 from v-viinde/bamboo

USER STORY 1137599 SaaS App Tutorial: SAML SSO for Bamboo by resoluti…

anilmur committed:
Updated date

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger18 committed:
Merge pull request #27508 from dksimpson/dean-cdn

Dean cdn

Lisaco88 committed:
Merge pull request #28821 from v-viinde/trelloupdate

USER STORY 1135553 SaaS App Tutorial: Trello Update

Lisaco88 committed:
Merge pull request #29335 from georgewallace/auto1082921

automation : updates to address content bug 1082921

v-alje committed:
Merge pull request #29338 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #29337 from anilmur/master

Added HEVC as an available output video codec
